src/Sarif.Validation/ResultLogBuilder.cs

```diff
@@ -85,7 +85,10 @@ public class ResultLogBuilder: IDisposable
                 true,           // Produce verbose output.
                 new[] { instanceFilePath, schemaFilePath },
                 false,          // Do not compute target hash.
-                null);          // The version of this tool has no prerelease info.
+                null,          // The version of this tool has no prerelease info.
+                null);        // no invocation tokens to redact
```

Comment:
... or you could use named parameters for documentation (`verbose: true`, _etc._)